spec/comment_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Comments, type: :model do
```

Comment:
There are some tests that do not pass. Kindly see the errors below and make sure all the tests are green

Reply:
Thank you @SajjadAhmad14 for you suggestion :pray: